socorro/external/es/crashstorage.py

```diff
@@ -17,6 +18,10 @@
 from socorro.external.crashstorage_base import Redactor
 
 
+FIELD_NAME_STRING_ERROR_REGEX = re.compile(r'field=\"([\w.]+)\"')
```

Comment:
What about underscores? Or hyphens?
You could consider perhaps...
```python
FIELD_NAME_STRING_ERROR_REGEX = re.compile('field="[^"]+"', re.DOTALL)
```
That way it's ANYTHING (even newlines and whitespace) it'll match just fine. Perhaps that not what you want but just a worthwhile thought.